aspnetcore/razor-pages/ui-class.md

```diff
@@ -221,3 +221,28 @@ Suppose *RazorUIClassLib/Pages/Shared* contains two partial files: *_Header.csht
   <partial name="_Footer">
 </body>
 ```
+
+## Create a Razor class library with static assets
+
+Razor class libraries (RCL) frequently require companion static assets that can be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available to a consuming app.
+
+To include companion assets as part of a Razor class library, create a *wwwroot* folder in the class library and include any required files in that folder.
+
+When packing a Razor class library, all companion assets in the *wwwroot* folder are included in the package automatically and are made available to apps referencing the package.
+
+### Consume content from a referenced Razor class library
+
+The files included in the *wwwroot* folder of the Razor class library are exposed to the consuming app under the prefix `_content/{LIBRARY NAME}/`. The consuming app references these assets via `<script>`, `<style>`, `<img>`, and other HTML tags.
```

Comment:
Also I think we should go ahead and doc that "." characters are removed from the library name as pointed out by @mikes-gh.